libs/search-widget/src/widgets/search-widget/SearchResults.svelte

```diff
@@ -83,7 +84,7 @@
     {:else if !$pendingResults && $smartResults.length === 0 && !$onlyAnswers}
       <strong>{$_('results.empty')}</strong>
     {:else}
-      {#if $hasPartialResults}
+      {#if $hasPartialResults || $hasChatPartialResults}
```

Comment:
here we want a new warning message saying `Sorry, the service is overloaded and could not deliver an answer.`